code/modules/reagents/reagent_containers.dm

```diff
@@ -24,6 +24,13 @@
 	if(list_reagents)
 		reagents.add_reagent_list(list_reagents)
 
+/obj/item/weapon/reagent_containers/CheckParts(list/content)
+	..()
+	var/datum/reagents/RS = new(volume)
+	for(var/datum/reagent/R in contents)

```

Comment:
No, all the things in list/content are getting transfered into contents on on the atom level on checkparts